app/src/androidTest/java/com/sdp/movemeet/view/activity/ListActivityTest.java

```diff
@@ -17,38 +17,11 @@
 
 import java.util.concurrent.CountDownLatch;
 
-@LargeTest
 @RunWith(AndroidJUnit4.class)
 public class ListActivityTest {
 
     private FirebaseAuth fAuth;
     private FirebaseDatabase database;
 
-    @Test
-    public void listOfActivityTest() {
-        CountDownLatch latch = new CountDownLatch(1);
-        fAuth = FirebaseAuth.getInstance();
-        fAuth.signInWithEmailAndPassword("movemeet@gmail.com", "password").addOnSuccessListener(new OnSuccessListener<AuthResult>() {
-            @Override
-            public void onSuccess(AuthResult authResult) {
-                latch.countDown();
-            }
-        }).addOnFailureListener(new OnFailureListener() {
-            @Override
-            public void onFailure(@NonNull Exception e) {
-                assert (false);
-            }
```

Comment:
Why did you remove this test? Is this code covered by some other test now?

Reply:
I empty the ActivityListActivity for the moment, because this activity do nothing and lower the test coverage

Reply:
Ok, that makes sense to me